Expensify/App
# Add Spanish locales

Please, let's not focus too much on the translations themselves, we can iterate on them later.

### Fixed Issues

Fixes https://github.com/Expensify/Expensify/issues/152156

### Tests
- Go to the JS console > Application > Local Storage and set a new key `preferredLocale` to `"es-ES"` (including the quotes)
- Play around the app and check texts are correctly translated to spanish
- Delete the preferredLocale if you don't want e.cash to remain in spanish forever

### Tested On


### Screenshots


#### Web
![image](https://user-images.githubusercontent.com/521248/120724597-729dff00-c4d4-11eb-8e9b-f7a5c53479b1.png)
![image](https://user-images.githubusercontent.com/521248/120724600-7467c280-c4d4-11eb-8ab0-b48b90a2d0a5.png)
![image](https://user-images.githubusercontent.com/521248/120724608-76318600-c4d4-11eb-9cd9-1f3dd284508f.png)
![image](https://user-images.githubusercontent.com/521248/120724614-77fb4980-c4d4-11eb-90b0-b76f2de53fe3.png)
